common/templates/common/blocks/info_table/_url.html

```diff
@@ -4,9 +4,14 @@
      <li class="list-table__row">
         <div class="anim-table-row list-table__item info-table--col-3 {% if item.image %}info-table--row-3 info-table--col-3-desktop{% endif %}">
             {% if item.image %}
-                <div class="list-table__col list-table__small_screen_only">
+                <div class="list-table__col info-table-row__hover-only">
                     <figure class="info-table__figure">
-                        {% image item.image fill-66x66 alt=item.title %}
+						{% image item.image fill-66x66 as small_img %}
+						{% image item.image fill-120x120 as large_img %}
```

Comment:
`fill-` will crop the image. Do we want the image to crop or scale to fit?

Reply:
Hmm, maybe we should change to scaling, as you have done in #1345? (i.e. replace `fill` with `width` here)

Reply:
Having done some testing, I think actually height is the correct thing to scale to, as the design expects a fixed height.

Reply:
Makes sense to me.